chia/__init__.py

```diff
@@ -1,5 +1,14 @@
+import multiprocessing
+
 from pkg_resources import DistributionNotFound, get_distribution, resource_filename
 
+start_method = "spawn"
+try:
+    multiprocessing.set_start_method(start_method)
+except RuntimeError:
+    if multiprocessing.get_start_method(allow_none=True) != start_method:
```

Comment:
do I understand this correctly; you want to ignore any runtime error from this call, *except* the ones where the "start method" was set. Is that a likely thing to happen?

I think it might warrant a comment if you expect some error here that would still leave us in a state we're happy with.
Without such use case, it would seem simpler to just leave this a plain call, and if it fails, the process dies.

Reply:
Yes, I'll add a comment.

I think the gist was that if it has already been set then you get an error setting regardless of setting it to the same thing or not.  I could check first then set if needed, unless I'm forgetting some nuance.  But I went this route that can be read as "try to set the start method, if there is an error then ignore it if the start method is what we want".

Note that on the test side of things I also set the start method since we have plugins using multiprocessing so even with this being an application we already have "other code" setting the start method before us.